spacy/pipeline/functions.py

```diff
@@ -65,3 +69,88 @@ def merge_subtokens(doc: Doc, label: str = "subtok") -> Doc:
         for span in spans:
             retokenizer.merge(span)
     return doc
+
+
+default_long_token_splitter_config = """
+[config]
+long_token_length = 25
+split_length = 10
+"""
+
+DEFAULT_LONG_TOKEN_SPLITTER_CONFIG = Config().from_str(
+    default_long_token_splitter_config
+)["config"]
+
+
+@Language.factory(
+    "long_token_splitter",
+    default_config=DEFAULT_LONG_TOKEN_SPLITTER_CONFIG,
```

Comment:
```suggestion
    default_config={"long_token_length": 25, "split_length": 10},
```

Maybe we can just inline this as a dict here? We haven't been entirely consistent with this but I think if the default config doesn't contain nested sections and is short enough, we shouldn't have to do the whole `Config.from_str` roundtrip.